third_party/terraform/website/docs/guides/version_3_upgrade.html.markdown

```diff
@@ -93,6 +93,55 @@ provider "google" {
 
 ## Resource: `google_container_cluster`
 
+### `ip_allocation_policy` will catch out-of-band changes, `use_ip_aliases` removed
+
+-> This change and "Automatic subnetwork creation for VPC-native clusters
+removed" are related; see the other entry for more details.
```

Comment:
Can you link to it?

Reply:
I have no idea what link it will turn into, so not really unfortunately. I can amend it once `2.20` is out.